bin/generate-tiles

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# For backward compatibility, allow both PG* and POSTGRES_* forms,
+# with the non-standard POSTGRES_* form taking precedence.
+# An error will be raised if neither form is given, except for the PGPORT
+export PGDATABASE="${POSTGRES_DB:-${PGDATABASE?}}"
+export PGUSER="${POSTGRES_USER:-${PGUSER?}}"
+export PGPASSWORD="${POSTGRES_PASSWORD:-${PGPASSWORD?}}"
+export PGPORT="${POSTGRES_PORT:-${PGPORT:-5432}}"
+
+# List of postgres servers
+# "xxx.xxx.xxx.xxx&host=xxx.xxx.xxx.xxx&host=..."
+if [[ -z "${PGHOSTS_LIST}" ]]
+then
+  export HOST_COUNT=1
+  export PGHOSTS="${POSTGRES_HOST:-${PGHOST?}}"
+else
+  export HOST_COUNT=`awk -F"&" '{print NF}' <<< "${PGHOSTS_LIST}"`
+  export PGHOSTS="${PGHOSTS_LIST}"
+fi
+
+export NO_GZIP=${NO_GZIP:-1}
+if [[ ${NO_GZIP} ]] && [[ ! ${NO_GZIP} == "0" ]] && [[ ! ${NO_GZIP} == "true" ]]
+then
+  export NO_GZIP=1
+fi
+
+export USE_KEY_COLUMN=${USE_KEY_COLUMN:-1}
+if [[ ${USE_KEY_COLUMN} ]] && [[ ! ${USE_KEY_COLUMN} == "0" ]] && [[ ! ${USE_KEY_COLUMN} == "true" ]]
+then
+  export USE_KEY_COLUMN=1
+fi
+
+export FUNC_ZXY=${FUNC_ZXY:-getmvt}
+export TEST_ON_STARTUP="${TEST_ON_STARTUP:-0}"
```

Comment:
Should the default be some tile here?  You could use the same one i use for default, e.g. `14/9268/3575`.  It's good to do a test on startup to catch any connection errors.  Note that an empty value is the same as `false`, whereas not setting it would enable it by default.

Reply:
pgquery's default  - `14/9268/3575` - didn't work for me because it is not covered by my download area.
Could `0/0/0` be an option here?

Reply:
`0/0/0` is not a good candidate because it could take over an hour to generate.  It is ok if the downloaded area is not including that tile -- as long as query runs ok, even if it returns an empty result, it shows that everything is working. Plus you can always configure it to test one of the tiles inside your boundaries.

Reply:
I had the same problem that the test tile wasn't inside my downloaded area. But as long it is not an issue and the query runs and returns empty result that serves only for testing I think we are good. I put `14/9268/3575` as default for now.